ui/raidboss/data/05-shb/eureka/zadnor.js

```diff
@@ -81,30 +133,1155 @@ export default {
+      condition: (data) => data.ce === 'diremite',
+      alertText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Away From Orbs',
+        },
+      },
+    },
     // ***** Here Comes the Cavalry *****
+    {
+      id: 'Zadnor Cavalry Gust Slash',
+      netRegex: NetRegexes.startsUsing({ source: 'Clibanarius', id: '5D7D' }),
+      condition: (data) => data.ce === 'cavalry',
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime) - 5,
+      response: Responses.knockback(),
+    },
+    {
+      id: 'Zadnor Cavalry Raw Steel',
+      netRegex: NetRegexes.startsUsing({ source: 'Clibanarius', id: '5D87' }),
+      condition: (data) => data.ce === 'cavalry',
+      response: (data, matches, output) => {
+        // cactbot-builtin-response
+        output.responseOutputStrings = {
+          avoidCharge: {
+            en: 'Avoid Charge',
+          },
+          runAway: {
+            en: 'Run Away From Boss',
+            de: 'Renn weg vom Boss',
+            fr: 'Courez loin du boss',
+            ja: 'ボスから離れる',
+            cn: '远离Boss',
+            ko: '보스와 거리 벌리기',
+          },
+        };
+
+        if (matches.target === data.me)
+          return { alertText: output.runAway() };
+        return { infoText: output.avoidCharge() };
+      },
+    },
+    {
+      id: 'Zadnor Cavalry Call Raze',
+      netRegex: NetRegexes.startsUsing({ source: 'Clibanarius', id: '5D8C', capture: false }),
+      condition: (data) => data.ce === 'cavalry',
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Zadnor Cavalry Magitek Blaster',
+      netRegex: NetRegexes.startsUsing({ source: 'Clibanarius', id: '5D90' }),
+      condition: (data) => data.ce === 'cavalry',
+      response: Responses.stackMarkerOn(),
+    },
     // ***** Head of the Snake *****
     // ***** There Would Be Blood *****
+    {
+      id: 'Zadnor Blood Cloud Of Locusts',
+      netRegex: NetRegexes.startsUsing({ source: 'Hanbi', id: '5C10', capture: false }),
+      condition: (data) => data.ce === 'blood',
+      response: Responses.getOut(),
+    },
+    {
+      id: 'Zadnor Blood Plague Of Locusts',
+      netRegex: NetRegexes.startsUsing({ source: 'Hanbi', id: '5C11', capture: false }),
+      condition: (data) => data.ce === 'blood',
+      response: Responses.getIn(),
+    },
+    {
+      id: 'Zadnor Blood Dread Wind',
+      netRegex: NetRegexes.startsUsing({ source: 'Hanbi', id: '5BAE', capture: false }),
+      condition: (data) => data.ce === 'blood',
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Zadnor Blood Gale Cannon',
+      netRegex: NetRegexes.startsUsing({ source: 'Hanbi', id: '53E3', capture: false }),
+      condition: (data) => data.ce === 'blood',
+      alertText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          // TODO: should this be a response/output?
+          en: 'Out of Front',
```

Comment:
I think the only existing output like this is Tsukuyomi's "Get out of front" for if you're a non-healer, non-tank, non-target for the Torment tankbuster, and I would argue that should probably be changed to tankCleave().
Does Ouputs.getBehind work better here? Or Brute Justice `Megabeamu~!`?

Reply:
This is a wide line aoe that's not targeted on the tank where back and sides are both safe.  I am torn, because the specificity is nice (if you're a caster who is planting).  I'm fine making this "get out of front" is this becomes an output.  Leaving for now unless you feel strongly?

Reply:
I definitely do not feel strongly here.